templates/reverse.j2

```diff
@@ -8,24 +8,24 @@ $TTL 1W
 	IN	NS	ns1.{{ dns.clusterid }}.{{ dns.domain | lower }}.
 ;
 ; syntax is "last octet" and the host must have fqdn with trailing dot
-1       IN      PTR     helper.{{ dns.clusterid }}.{{ dns.domain }}.
+{{ helper.ipaddr.split('.')[3] }}	IN      PTR     helper.{{ dns.clusterid }}.{{ dns.domain | lower }}.
 
 {% for m in masters %}
-{{ m.ipaddr.split('.')[3] }}	IN	PTR	{{ m.name | lower }}.{{ dns.clusterid }}.{{ dns.domain | lower }}.
+{{ m.ipaddr.split('.')[3] }}	IN	PTR	{{ m.name | lower }}.
```

Comment:
This won't work. it needs the fqdn if using the `.` delimiter.

Reply:
Discovered in my install that m.name is a FQDN.

Reply:
This might be specific to your env. I'm guess you put the FQDN in your `vars.yaml` file. This isn't true for all cases.

Reply:
Mmm, good point. It actually comes from the terraform scripts which create the nodes in OpenStack/PowerVC. I will make the change over there.